### PR TITLE
Storing stocksfeed getting from Stock_exchange_engine

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: internal/services/order/stocks_feed.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		newStock:=model.StocksFeed{
+			StockName: stock.StockName,
+			LTP:       int(stock.LTP),
+			Open:      int(stock.Open),
+			High:      int(stock.High),
+			Low:       int(stock.Low),
+			TradedAt:  stock.UpdatedAt,
+			CreatedAt: time.Now(),
+		}
+		if err=config.DB.Create(newStock).Error;err!=nil{
```
Comment (ghost) [inline internal/services/order/stocks_feed.go]: table

File: internal/services/order/stocks_feed.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	for _,stock:=range res{
+		newStock:=model.StocksFeed{
+			StockName: stock.StockName,
+			LTP:       stock.LTP,
+			Open:      stock.Open,
+			High:      stock.High,
+			Low:       stock.Low,
+			TradedAt:  stock.UpdatedAt,
+		}
+		if err=config.DB.Table("stocks_feed").Create(&newStock).Error;err!=nil{
```
Comment (mukesh0513) [inline internal/services/order/stocks_feed.go]: are we not updating stock feed even if one fails?

File: internal/services/order/stocks_feed.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package order
```
Comment (mukesh0513) [inline internal/services/order/stocks_feed.go:1]: why this method is in order service?

File: internal/services/order/order_rest.go [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+				if err=config.DB.Table("holdings").Where("id",check.Id).Update(check).Error;err!=nil{
+					return err
+				}
+			}
+		}
+		if err=config.DB.Table("pending_orders").Where("order_id=?",res.OrderID).Delete(p).Error;err!=nil{
+			return err
+		}
+	} else if res.Status=="Half Completed"{
```
Comment (mukesh0513) [inline internal/services/order/order_rest.go]: statuses can't be space separated strings. Maintain statuses as constants.

File: internal/services/order/stocks_feed.go (hunk 2 above)
Comment (ghost) [reply internal/services/order/stocks_feed.go]: Added continue instead of return

File: internal/services/order/stocks_feed.go (hunk 3 above)
Comment (ghost) [reply internal/services/order/stocks_feed.go:1]: then can we keep it as a separate file itself in the services folder?

File: internal/services/order/stocks_feed.go [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+		newStock:=model.StocksFeed{
+			StockName: stock.StockName,
+			LTP:       stock.LTP,
+			Open:      stock.Open,
+			High:      stock.High,
+			Low:       stock.Low,
+			TradedAt:  stock.UpdatedAt,
+		}
+		if err=config.DB.Table("stocks_feed").Create(&newStock).Error;err!=nil{
+			log.Println("Error in Creating a stock field")
```
Comment (mukesh0513) [inline internal/services/order/stocks_feed.go]: can we log stock name too.

File: internal/services/order/stocks_feed.go (hunk 3 above)
Comment (mukesh0513) [reply internal/services/order/stocks_feed.go:1]: oh.. you have named transaction engine as order service. Is that so?

**Comment by mukesh0513** [comment]
Do go gmt project.

**Review by mukesh0513 — APPROVED** [review]
LGTM

**Pull request merged**